docs/assets/js/_partials/chart-area.js

```diff
@@ -0,0 +1,251 @@
+class AreaChart
+{
+    constructor(config, canvas, actionBar)
+    {
+        this.config = config;
+        this.actionBar = actionBar;
+
+        this.context = canvas.getContext('2d');
+        this.views = [];
+        this.defaultView = null;
+
+        // Show spinning animation while loading
+        this.spinner = createSpinner(canvas);
+    }
+
+    render(data)
+    {
+        sortTimeData(data);
+
+        this._checkData(data);
+        this._initializeViews(data);
+
+        // Build chart data for each view separately
+        for (const view of this.views)
+        {
+            this._aggregateData(view);
+
+            // Build the actual data used by Chart.js for rendering
+            this._buildChartData(view);
+        }
+
+        this.chartJSChart = new Chart(
+            this.context,
+            {
+                type: 'line',
```

Comment:
Interesting that the type is still `line` although it's an area chart.

Reply:
Yep. I guess the Chart.js developers wanted to kill as many birds with one stone as possible :smile:.